repos/system_upgrade/el7toel8/actors/cloud/grubenvtofile/libraries/grubenvtofile.py

```diff
@@ -0,0 +1,20 @@
+from leapp.libraries.stdlib import api, run, CalledProcessError
+
+
+BIOS_PATH = '/boot/grub2/grubenv'
+EFI_PATH = '/boot/efi/EFI/redhat/grubenv'
+
+
+def grubenv_to_file():
+    try:
+        run(['unlink', BIOS_PATH])
+    except CalledProcessError as err:
+        api.current_logger().warning('Could not unlink {}: {}'.format(BIOS_PATH, str(err)))
+        return
+    try:
+        run(['cp', EFI_PATH, BIOS_PATH])
```

Comment:
Maybe to be sure, this could be better:
```suggestion
        run(['cp', '-a', EFI_PATH, BIOS_PATH])
```

Reply:
I agree. Originally wanted to do but as the metadata were copied ok...